test/manipulation.test.js

```diff
@@ -889,6 +897,51 @@ describe('manipulation', function() {
     });
   });
 
+  describe('updateOrCreate when forceId is true', function() {
+    var Post;
+    before(function definePostModel(done) {
+      var ds = getSchema();
+      Post = ds.define('Post', {
+        title: {type: String, length: 255},
+        content: {type: String},
+      }, {forceId: true});
+      ds.automigrate('Post', done);
+    });
+
+    it('fails when id does not exist in db', function(done) {
+      var post = {id: 123, title: 'a', content: 'AAA'};
```

Comment:
Hard-coded ids like `123` don't work with the Cassandra connector, please check the recent changes my by @Setogit and do the same here.